dask_glm/algorithms.py

```diff
@@ -97,7 +97,7 @@ def gradient_descent(X, y, max_iter=100, tol=1e-14, family=Logistic, **kwargs):
     stepSize = 1.0
     recalcRate = 10
     backtrackMult = firstBacktrackMult
-    beta = np.zeros(p)
+    beta = np.zeros_like(X._meta, shape=(p))
```

Comment:
```suggestion
    beta = np.zeros_like(X._meta, shape=p)
```

I think that we probably don't need the extra parens around `p` here and below